.dependencies.ln

```diff
@@ -0,0 +1,5 @@
+from @std/deps import Package, install, add, commit, dependency, using, block
+
+on install fn (package: Package) = package
+  .using(['@std/app', '@std/cmd', '@std/root', '@std/seq'])
```

Comment:
Everything gets `root` implicit in every module,  so we don't need to call it out.